From 7fc838cca334ccf3d388c5598ff7fae88dbe4513 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 25 Sep 2012 12:15:19 +0100
Subject: [PATCH] test-launch-race: Add SELinux label to $TMPDIR.

This is a temporary workaround for
https://bugzilla.redhat.com/show_bug.cgi?id=860235
which can be removed once this bug is fixed.
---
 tests/protocol/test-launch-race.pl |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/tests/protocol/test-launch-race.pl b/tests/protocol/test-launch-race.pl
index f933bfe..7e1330b 100755
--- a/tests/protocol/test-launch-race.pl
+++ b/tests/protocol/test-launch-race.pl
@@ -32,6 +32,11 @@ exit 77 if $ENV{SKIP_TEST_LAUNCH_RACE_PL};
 my $tmpdir = tempdir (CLEANUP => 1);
 $ENV{TMPDIR} = $tmpdir;
 
+# Temporary workaround for RHBZ#860235 which can be removed
+# once that bug is fixed.
+system ("chcon system_u:object_r:tmp_t:s0 $tmpdir") == 0
+    or warn "chcon: $tmpdir: $!";
+
 my $pid = fork();
 die ("fork failed: $!") if ($pid < 0);
 
-- 
1.7.10.4

